commander/src/commander-plugin.c

```diff
@@ -796,8 +792,32 @@ plugin_cleanup (void)
   }
 }
 
-void
-plugin_help (void)
+
+static void
+plugin_commander_help (G_GNUC_UNUSED GeanyPlugin *plugin, G_GNUC_UNUSED gpointer pdat)
+{
+  utils_open_browser("https://plugins.geany.org/commander.html");
```

Comment:
style: space before the open parenthesis

Reply:
Why changing the target URI?  Why open a website when the same documentation is available locally?